compiler/damlc/lib/DA/Cli/Args.hs

```diff
@@ -12,22 +12,18 @@ import Options.Applicative hiding (customExecParser, execParser)
 import Options.Applicative.Internal
 import Options.Applicative.BashCompletion
 import Options.Applicative.Common
-import System.Environment
 import System.IO
 
 -- | Runs the argument parser in a normal way unless the first argument is "lax"
 --   In this case it discards invalid flags and continues
-execParserLax :: Op.ParserInfo a -> IO a
-execParserLax = customExecParser Op.defaultPrefs
-
-customExecParser :: Op.ParserPrefs -> Op.ParserInfo a -> IO a
-customExecParser pprefs pinfo = do
-  args <- getArgs
+execParserLax :: [String] -> Op.ParserInfo a -> IO a
+execParserLax args pinfo = do
+  putStrLn $ "trace: " <> show args
```

Comment:
Looks like this is leftover from debugging?

Reply:
It is, thanks for spotting. Removed.